vpd-tool/src/vpd_tool_main.cpp

```diff
@@ -37,13 +48,23 @@ int main(int argc, char** argv)
     auto l_fileOption = l_app.add_option("--file", l_filePath,
                                          "Absolute file path");
 
+    auto l_keywordValueOption =
+        l_app.add_option("--value, -V", l_keywordValue,
+                         "Keyword value in ascii/hex format."
+                         " ascii ex: 01234; hex ex: 0x30313233");
+
+    auto l_hardwareFlag = l_app.add_flag("--Hardware, -H",
+                                         "CAUTION: Developer only option.");
+
     auto l_readFlag = l_app.add_flag("--readKeyword, -r", "Read keyword")
                           ->needs(l_objectOption)
                           ->needs(l_recordOption)
                           ->needs(l_keywordOption);
 
-    auto l_hardwareFlag = l_app.add_flag("--Hardware, -H",
```

Comment:
why do you want to move auto l_hardwareFlag to line 56? these are causing additional differences to note between merged code and your PR which are not needed.
Try to have minimal required changes in your PR.

Reply:
Please take a look of commit message, options are arranged properly.

The changes includes adding new option, so we shouldn't worry about keeping minimal changes, until we are making related changes.

Reply:
what I meant is moving a piece of code here and there without any valid reason is causing additional differences, which are not related and not required.

Reply:
Here we are implementing options one by one, so new option can go and sit in appropriate place. no harm in here .